frontend/src/app/typeof.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { TypeOfPipe } from './typeof.pipe';
+
+describe('TypeOfPipe', () => {
+  it('create an instance', () => {
+    const pipe = new TypeOfPipe();
+    expect(pipe).toBeTruthy();
```

Comment:
**suggestion:** add a few test cases for different types of objects. Though, in general I would argue this kind of a pipe is often code smell as you shouldn't rely on types of things on templates